managed/services/dbaas/kubernetes/kubernetes.go

```diff
@@ -742,7 +742,7 @@ func (k *Kubernetes) InstallOLMOperator(ctx context.Context) error {
 		return errors.Wrap(err, "cannot decode olm resources")
 	}
 
-	resources := append(crdResources, olmResources...)
+	resources := append(crdResources, olmResources...) //nolint:gocritic
```

Comment:
what was the linter complaining about here?

Reply:
It's claiming we should be appending to the same slice, 
i.e. `crdResources := append(crdResources, olmResources...)`